Shop-Facade-Open-Api-Spec.yaml

```diff
@@ -0,0 +1,1108 @@
+            \ delivery date), must be today's date or later. (Note: The time element\
+            \ is stored but not displayed in e-Navigator). Format: UTC YYYY-MM-DDThh:mm:ss\
+            \ E.g. 2012-1102T14:28:37.6613351+00:00 (Time and offset optional)."
+          format: date-time
+        deliveryMethod:
+          type: string
+          description: "Required. Permits and/or activation keys are provided either\
+            \ as downloads through GetOrder service or by email. The possible values:\
+            \ Email, Download, or Both."
+          enum:
+          - Email
+          - Download
+          - Both
+        deliveryAddress:
+          maxLength: 255
+          type: string
+          description: Not required on "digital" only orders. Optional on orders including
+            "paper" products. A delivery address of an order.
+        orderItems:
+          type: array
+          description: Required. The items in the order.
+          items:
+            $ref: '#/components/schemas/orderItems'
+    orderItems:
+      required:
+      - productId
+      - productType
+      type: object
+      properties:
+        productType:
+          type: string
+          description: Required. The product type of the product to be ordered.
+          enum:
+          - S101
+          - S102
+          - S104
+          - S111
+          - S122
+        productId:
+          type: string
+          description: "Required. The product ID of the product to be ordered from\
+            \ the current S100 catalogue, of the relevant product type. NOTE:\
+            \ this is case sensitive."
+        productPeriod:
+          type: integer
+          description: "Required for S100 Charts. The period the product is required\
+            \ for in months. Accepts only: 3, 6, 9, or 12."
+        quantity:
+          type: integer
+          description: Required for "paper" products. Not required for other products.
+            The quantity of product required.
+    submitOrderResponse:
+      type: object
+      properties:
+        orderId:
+          type: integer
+        accountId:
+          type: integer
+        messages:
+          type: array
+          items:
+            $ref: '#/components/schemas/responseMessage'
+        orderStatus:
+          type: array
+          items:
+            $ref: '#/components/schemas/orderItemStatus'
+    orderItemStatus:
+      type: object
+      properties:
+        product:
+          type: string
+        status:
+          type: string
+    responseMessage:
+      type: string
+    getHoldingsResponse:
+      type: array
+      items:
+        $ref: '#/components/schemas/holding'
+    holding:
+      type: object
+      properties:
+        productCode:
+          type: string
+        title:
+          type: string
+        productType:
+          type: string
+          enum:
+          - S101
+          - S102
+          - S104
+          - S111
+          - S122
```

Comment:
S122 isn't in the initial S-100 offering? Either this needs to only have S101, S102, S104 & S111, or it needs to allow all potential future S100 standards.

Reply:
removed s122.